test/junitTrace/JUnitTrace.gmk

```diff
@@ -26,17 +26,17 @@
 #----------------------------------------------------------------------
 
 $(BUILDTESTDIR)/JUnitTrace.ok: \
-	    $(JTREG_IMAGEDIR)/lib/jtreg.jar \
-	    $(JTREG_IMAGEDIR)/bin/jtreg
+		$(JTREG_IMAGEDIR)/lib/jtreg.jar \
+		$(JTREG_IMAGEDIR)/bin/jtreg
 	$(RM) $(@:%.ok=%) ; $(MKDIR) $(@:%.ok=%)
 	$(JTREG_IMAGEDIR)/bin/jtreg $(JTREG_OPTS) \
 		-w:$(@:%.ok=%)/work -r:$(@:%.ok=%)/report \
 		-jdk:$(JDKHOME) \
 		-agentvm \
```

Comment:
It might be better to upgrade this test to run in both modes (othervm, agentvm(), although maybe that could be could separately

Reply:
I agree — should be done in separate PR to keep the changes of this PR focussed around making the JUnit Platform Launcher work.

Reply:
Done via ...